p/themes/base-theme/template.css

```diff
@@ -345,7 +346,7 @@ a.btn {
 /*=== Tree */
 .tree {
 	margin: 0;
-	padding: 0 0 15em 0;
+	padding: 0;
```

Comment:
What was the reason for this change?
https://github.com/FreshRSS/FreshRSS/pull/1489